codes/1_planning.py

```
@@ -13,13 +15,14 @@
 args    = parser.parse_args()
 
-client = OpenAI(api_key = os.environ["OPENAI_API_KEY"])
 
 paper_name = args.paper_name
 gpt_version = args.gpt_version
 pdf_json_path = args.pdf_json_path
 output_dir = args.output_dir
 
+agent = Agent(gpt_version)
+
 with open(f'{pdf_json_path}') as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)